model/lifecycle.go

```diff
@@ -294,18 +298,21 @@ func SetVersionsPriority(ctx context.Context, versionIds []string, priority int6
 		return errors.Wrap(err, "setting priority for versions")
 	}
 
-	// negative priority - these tasks should never run, so unschedule now
+	tasks, err := task.FindAll(db.Query(query))
+	if err != nil {
+		return errors.Wrap(err, "getting tasks for versions")
+	}
+	var taskIds []string
+	for _, t := range tasks {
+		taskIds = append(taskIds, t.Id)
+	}
+	event.LogManyTaskPriority(taskIds, caller, priority)
+
+	// Deactivate tasks with negative priority
```

Comment:
Can we change this comment (and the other one that is identical) back to something closer to the original comment, like `Tasks with negative priority should never run, so we unschedule them`